### ISSUE TITLE
[Feature Request] Custom Autocomplete Dictionary -> Ability to define sets of words

### ISSUE THREAD

**Issue by FelipeRearden** [tadashi-aikawa/obsidian-various-complements-plugin#13]
Hello @tadashi-aikawa !!!!

I just saw the new Custom Dictionary and I wanna request an additional feature :) 

I must say that I'm not an English native speaker. Please let me know if you didnt understood something !

## FR

This is my `dictionary.txt` file with all my words

```
White
Blue
Red
Car
Plane
Ship
Apple
Orange
Grape
```
My FR is a new feature to create subsets/groups for words inside the  `dictionary.txt` file

```
{color}White
{color}Blue
{color}Red
{transp}Car
{transp}Plane
{transp}Ship
{fruit}Apple
{fruit}Orange
{fruit}Grape
```

## Example

Following my example above. 

Step 1 - When starting to type `col`the context menu will only show `White` `Blue` `Red`
Step 2 - After selecting `Blue` the word `col` will be replace by the word blue 

---

I don't know if what I am suggesting is feasible but will be amazing for me to be able to define different words using subset/groups

Let me know if you need more information!!!

Thanks for reading this! 

I wish you a fantastic day!

**Comment by tadashi-aikawa** [comment]
Hello, @FelipeRearden.

Thank you for your Feature Request and kind description!  Don't worry, I'm not an English native speaker, too 😄 

> My FR is a new feature to create subsets/groups for words inside the dictionary.txt file

It might be a bit difficult to use subsets/groups; instead, I want to implement `alias` as follows. ( `<TAB>` is a delimiter)

```
White<TAB>color
Blue<TAB>color
Red<TAB>color
Car<TAB>transp
Plane<TAB>transp
Ship<TAB>transp
Apple<TAB>fruit
Orange<TAB>fruit
Grape<TAB>fruit
```

Does this seem to satisfy your request?

**Comment by FelipeRearden** [comment]
Hello @tadashi-aikawa !!!!

> Thank you for your Feature Request and kind description!  Don't worry, I'm not an English native speaker, too 😉 

Thanks :) I'm glad that you could understood what I wrote :)

> It might be a bit difficult to use subsets/groups; instead, I want to implement `alias` as follows. ( `<TAB>` is a delimiter)

Great!@@@@ That exactly the word that I was trying to remember `alias` when I was creating this issue :)

> Does this seem to satisfy your request?

100% I'm so happy that I will be able to create alias for group kinds of words :)


Thanks again! I wish you a great day!

**Comment by tadashi-aikawa** [comment]
@FelipeRearden 
I have released [v1.4.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/1.4.0) 🚀 

Would you please confirm it?

**Comment by FelipeRearden** [comment]
> @FelipeRearden 
> 
> I have released [v1.4.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/1.4.0) 🚀 
> 
> 
> 
> Would you please confirm it?

Thank you very much @tadashi-aikawa !!!!!!!!

I'm testing version `0.2.0` and I must ask what is the delimiter to define an alias? Is it the tab keyboard button ?

As soon I get your message I finish my text and I give my feedback to you :)

Thanks :)

**Comment by FelipeRearden** [comment]
> @FelipeRearden 
> 
> I have released [v1.4.0](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/releases/tag/1.4.0) 🚀 
> 
> 
> 
> Would you please confirm it?

Hello @tadashi-aikawa !!!!!

Thank you very much for this feature !!!!!!!!!!

Working perfect on version `0.2.0` Obsidian `0.12.19`


I'm very happy !!!!!

I wish you a fantastic day!
